src/data.rs

```diff
@@ -1,6 +1,8 @@
 use std::error::Error;
 use std::io::ErrorKind;
-use std::{env, fs, path::Path};
+use std::{fs, path::Path};
+
+use directories::{BaseDirs, ProjectDirs};
```

Comment:
```suggestion
use directories::BaseDirs;
```

(Assuming you take a related change below, too)

Reply:
Nevermind on this specific change.